Updating helm chart for stress-test-addons (#2223)
Azure/azure-sdk-tools: tools/stress-cluster/cluster/README.md

   * [Test Cluster](#test-cluster)
   * [Prod Cluster](#prod-cluster)
   * [Local Cluster](#local-cluster)

> * [Stress Test Addons](#stress-test-addons)

Reviewer: ```suggestion
* [Deploying Stress Test Addons](#deploying-stress-test-addons)
```

kind create cluster
```

> # Stress test addons

Reviewer: ```suggestion
# Deploying Stress Test Addons
```

Steps for deploying the stress test addons helm chart:

> 1. Increment the version number in Chart.yaml (e.g. 0.1.0 -> 0.1.1)

Reviewer: I would link to the actual filepath here in github.

> 1. Run ./deploy.ps1

Reviewer: Same as above (link).

> 1. Update all the helm chart versions for stress-test-addons refernces in stress-cluster/chaos/examples/

Reviewer: ```suggestion
1. Update all the helm chart versions for stress-test-addons dependency references in stress-cluster/chaos/examples/**/Chart.yaml
```

> 1. Run azure-sdk-tools\eng\common\scripts\stress-testing\deploy-stress-tests.ps1 script in the examples directory, this will update all the nested helm charts (-login tag is needed for the first run)

Reviewer: Github link to the examples directory here. Also, add that one should validate the tests succeeded (e.g. wait for pod to reach Running/Completed state vs. Error state).

Reviewer: And then, another list item here to specify to update all the stress tests across the other repos in the same manner.

kind create cluster
```

# Deploying Stress Test Addons
Steps for deploying the stress test addons helm chart:
1. Increment the version number in [Chart.yaml](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/kubernetes/stress-test-addons/Chart.yaml) (e.g. 0.1.0 -> 0.1.1).

> 1. Run [deploy.ps1](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/kubernetes/stress-test-addons/deploy.sh).

Reviewer: ```suggestion
1. Run [deploy.ps1](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/kubernetes/stress-test-addons/deploy.ps1).
```

> 1. Update all the helm chart versions for stress-test-addons dependency references in [stress-cluster/chaos/examples/](https://github.com/Azure/azure-sdk-tools/tree/main/tools/stress-cluster/chaos/examples)**/Chart.yaml.

Reviewer: It's not a valid link so I think this should just be represented as a file path.

```suggestion
1. Update all the helm chart versions for stress-test-addons dependency references in `azure-sdk-tools/tree/main/tools/stress-cluster/chaos/examples)**/Chart.yaml`.
```

>    1. Please verify that the tests have succeeded with each pod to reaching Running/Completed state.

Reviewer: Show the command(s) to run, e.g. deploy from examples + `kubectl get pods -w -n examples`.

> 1. Run azure-sdk-tools\eng\common\scripts\stress-testing\deploy-stress-tests.ps1 script in the [examples](https://github.com/Azure/azure-sdk-tools/tree/main/tools/stress-cluster/chaos/examples) directory, this will update all the nested helm charts (-login tag is needed for the first run).

Reviewer: This step should probably be combined with the above sub-step, as it's the explicit instruction for what the above step says to do at a high level. Also I would show the command to verify everything is working (e.g. `kubectl get pods -n examples -w` and look for Running/Completed no Error, etc.)

> 1. Update all the stress tests across the other repos in the same manner.

Reviewer: Stress test Chart.yaml files, be specific here.
